tests/data/comments6.py

```diff
@@ -0,0 +1,56 @@
+# test type comments
+def f(a, b, c, d, e, f, g, h, i):
+    # type: (int, int, int, int, int, int, int, int, int) -> None
+    pass
+
+
+def f(
+    a,  # type: int
```

Comment:
Can you also test this with other kinds of arguments (those with defaults, *args, **kwargs, etc.)?